docs/website/mkdocs.yml

```diff
@@ -63,9 +63,12 @@ extra:
     - icon: fontawesome/brands/discord
       link: https://discord.gg/wEWh6Z9nMU
       name: IREE Discord Server
+    - icon: fontawesome/solid/envelope
+      link: https://lists.lfaidata.foundation/g/iree-announce
+      name: IREE Announcement Mailing List
     - icon: fontawesome/solid/users
-      link: https://groups.google.com/forum/#!forum/iree-discuss
-      name: IREE Discuss Google Group
+      link: https://lists.lfaidata.foundation/g/iree-technical-discussion
+      name: IREE Technical Discussion Mailing List
```

Comment:
Similar to my other comment. These appear in the page footer.

envelope:
![image](https://github.com/user-attachments/assets/6ec12344-6fa6-4e5d-ad56-2861ce4b2308)

bullhorn:
![image](https://github.com/user-attachments/assets/d4fdf4a3-0a4b-456e-b272-fad9be48429a)

Reply:
With the above, this would become
![image](https://github.com/user-attachments/assets/f987773a-71a7-471d-9949-ecc0489cfcc2)

Reply:
Any variation there is fine with me, as long as it is consistent across the body text and the footer